### DISCUSSION TITLE
Migrate NextJS only app to Solito?

### DISCUSSION THREAD

**Discussion by gascenciom1998** [nandorojo/solito#225] [Q&A]
What's the easiest / quickest / recommended way to migrate from a NextJS only app to Solito? Is it required to maintain feature parity between next and expo to migrate or is it possible to migrate progressively?

**Comment by nandorojo** [comment]
you can migrate as progressively as you’d like by using platform-specific code at a file level, page level, or even within particular functions.

**Reply by gascenciom1998** [reply]
Got it thanks. And how can I have the expo app access endpoints on `pages/api` ?
